lib/activerecord/delay_touching.rb

```diff
@@ -6,9 +6,9 @@ module DelayTouching
     extend ActiveSupport::Concern
 
     # Override ActiveRecord::Base#touch.
-    def touch(name = nil)
+    def touch(*names)
       if self.class.delay_touching? && !try(:no_touching?)
-        DelayTouching.add_record(self, name)
+        DelayTouching.add_record(self, names)
```

Comment:
actually, you've just made me realize something that might simplify this thing. i'll post back shortly